src/v3/src/types/schema.ts

```diff
@@ -23,6 +23,15 @@ import { FunctionComponent } from 'preact';
 import { PasswordSettings } from './password';
 import { UserInfo } from './userInfo';
 
+export type FormBag = {
+  schema: Record<string, unknown>;
+  uischema: UISchemaLayout;
+  data: Record<string, unknown>;
+  // temp schema bag to handle client validation and form submission
+  dataSchema: Record<string, DataSchema>;
```

Comment:
From my previous comments we can just use schema and forego introducing a new custom field and type to keep things simple and easily scalable.

Reply:
cc: @lesterchoi-okta